auth_otp/settings_sample.py

```
@@ -0,0 +1,216 @@
+    'cleanup-expired-tokens': {
+        'task': 'auth_otp.tasks.cleanup_expired_tokens',
+        'schedule': crontab(hour=3, minute=30),  # هر روز ساعت 3:30 صبح
+    },
+}
+
+# ==================================
+# Middleware
+# ==================================
+
+MIDDLEWARE = [
+    # ... سایر middleware ها
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)